MapboxSearch/offline/src/androidTest/java/com/mapbox/search/offline/OfflineSearchEngineIntegrationTest.kt

```diff
@@ -306,13 +306,13 @@ internal class OfflineSearchEngineIntegrationTest {
         val results = searchEngineResult.requireResults()
         assertTrue(results.size > 5)
 
-        val result = results.first()
-        assertTrue(
-            compareWithApproximateLocations(
-                TEST_SEARCH_RESULT_MAPBOX,
-                result.rawSearchResult
-            )
-        )
+//        val result = results.first()
+//        assertTrue(
+//            compareWithApproximateLocations(
```

Comment:
Are these three commented assertions no longer relevant and safe to remove? Or do they need to be adjusted and added?